### PR TITLE
changelog: first approach

### PR DESCRIPTION
**Pull request by retrofox**
This PR adds a new block to the Jetpack extension blocks: `changelog`.

#### Changes proposed in this Pull Request:

* Add changelog block

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Start to create/edit a post
* Add the `changelog` block

You can change the label using the dropdown.

Screen Shot 2020-11-24 at 10 53 49 AM

There are three default values, and also it's possible to set a custom one.

It's possible to add an optional timestamp.

* save the post
* check that the content is properly rendered in the front-end.

#### Proposed changelog entry for your changes:

*

### CODE REVIEW

**Comment by jeherve** [comment]
Do you think you could post on +jetpackp2 about the feature, so Product can weigh in on whether this would be a good fit for the Jetpack plugin or not?

If there is already a discussion about this somewhere, I may have missed it, sorry! Could you add a link to it in the PR description?

Thank you!

File: extensions/blocks/changelog/components/time-stamp-control.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	BaseControl,
+	__experimentalNumberControl as NumberControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+function checkValidValue( val, max ) {
+	if ( val > max ) {
+		return val = max;
+	} else if ( val < 0 ) {
+		return val = 0;
```
Comment (mreishus) [inline extensions/blocks/changelog/components/time-stamp-control.js:14]: `=` or `==`? Also 2 lines above

Comment (retrofox) [reply extensions/blocks/changelog/components/time-stamp-control.js:14]: it should be just returned `0`. Updating. Thanks :-)

Comment (retrofox) [reply extensions/blocks/changelog/components/time-stamp-control.js]: Updated here https://github.com/Automattic/jetpack/pull/17885/commits/035c9626624f0076793abb393626b55669190009

**Comment by retrofox** [comment]
let's remove the timestamp from here

**Comment by creativecoder** [comment]
@retrofox Should we close this, since there's a changelog block already available in P2?

**Pull request closed without merging**